app/plugins/auth.js

```diff
@@ -0,0 +1,16 @@
+const { jwtConfig } = require('../config')
+const { validate } = require('../auth')
+
+module.exports = {
+  plugin: {
+    name: 'auth',
+    register: (server, options) => {
+      server.auth.strategy('jwt', 'jwt', {
+        key: jwtConfig.secret,
+        validate,
+        cookieKey: 'ffc_sfi_identity'
+      })
+      server.auth.default({ strategy: 'jwt', mode: 'try' })
```

Comment:
What if instead of using `try` we use `required`?  Then all routes by default will require authentication unless we disable it on that route.

As 99% of our journey needs authentication, it would save us having to add the `auth` on every route and only for those that don't need auth.

You could then remove all the `auth: { strategy: 'jwt' },` from the other routes.